lib/complate.js

```diff
@@ -0,0 +1,44 @@
+let { VirtualBundle } = require("faucet-pipeline-js/lib/bundle/virtual");
+
+// a plugin loads the highlighter for the language
+let loadLanguages = require("prismjs/components/");
+loadLanguages(["jsx"]);
+
+let rendering = `
+import Renderer, { Fragment as _F, createElement as _h } from "complate-stream";
```

Comment:
FWIW, I'd opted to obfuscate these so that users can import `Fragment` within snippets without running into arcane "already defined" errors. Might be worth adding a comment to that effect.